### ISSUE TITLE
Support for Google connector?

### ISSUE THREAD

**Issue by philippe-boyd-3r** [concourse/dex#11]
Concourse's Dex is way behind the upstream (328 commits behind dexidp:master). Is there any plans to put it up to date and have the ability to connect with the Google connector?

**Comment by piroux** [comment]
@vito @jwntrs 

I just saw the new tag `v2.25.0+fork.20201001161429` which includes all the commits of the latest release of the upstream `dex`.
:arrow_right:  Does it mean that it will be that version of `concourse/dex` which will be used in the next `6.6` concourse release ? 

Indeed, that would be great because it would bring direct support for the google connector. :tada: 

If this is not possible to use `v2.25.0+fork.20201001161429` in the upcoming `6.6` concourse release, what are the technical blocking points ? :clipboard:

**Comment by chenbh** [comment]
FWIW the current setup is that there's now an automated pipeline that rebases and releases a new concourse/dex whenever upstream dex does a release (https://github.com/concourse/concourse/issues/6007). The idea is that we would release concourse/dex under some version that gomod can consume, and use dependabot to update the go.mod on concourse/concourse.

However we still need to refactor the concourse/concourse codebase to use the upstream dex code (lots of stuff changed since our fork from last year), and that's tracked in https://github.com/concourse/concourse/issues/6128.

**tldr**: still not using upstream dex just yet, but we're likely to pick it up in the next couple weeks
